package.json

```diff
@@ -72,12 +82,16 @@
     "rimraf": "^3.0.0",
     "storybook-chromatic": "^3.4.1",
     "svg-to-jsx": "^1.0.2",
-    "svgo": "^1.3.2"
+    "svgo": "^1.3.2",
+    "typescript": "^3.7.3"
   },
   "scripts": {
     "clean": "rimraf dist",
-    "build": "yarn clean && cpx 'src/Styles/**/*' dist/Styles && cpx 'src/Components/**/*.{sass,scss}' dist/Components && babel src/index.js -d dist/ && babel src/Components -d dist/Components && babel src/Utils -d dist/Utils && babel src/Types -d dist/Types && babel src/Hooks -d dist/Hooks",
+    "build": "yarn clean && yarn build:types && yarn build:js",
+    "build:types": "tsc",
```

Comment:
be choosy

Reply:
This is only for type checking. The only "build" should happen in babel as before